pkg/didcomm/crypto/jwe/authcrypt/authcrypt.go

```diff
@@ -30,25 +33,14 @@ const XC20P = ContentEncryption("XC20P") // XChacha20 encryption + Poly1305 auth
 //nolint:gochecknoglobals
 var randReader = rand.Reader
 
-// errEmptyRecipients is used when recipients list is empty
-var errEmptyRecipients = errors.New("empty recipients")
-
-// errInvalidKeypair is used when a keypair is invalid
-var errInvalidKeypair = errors.New("invalid keypair")
-
-// errInvalidKey is used when a key is invalid
-var errInvalidKey = errors.New("invalid key")
-
-// errRecipientNotFound is used when a recipient is not found
-var errRecipientNotFound = errors.New("recipient not found")
-
 // errUnsupportedAlg is used when a bad encryption algorithm is used
 var errUnsupportedAlg = errors.New("algorithm not supported")
 
-// Crypter represents an Authcrypt Encrypter (Decrypter) that outputs/reads JWE envelopes
-type Crypter struct {
+// BaseCrypter represents an Authcrypt Encrypter (Decrypter) that outputs/reads JWE envelopes
```

Comment:
is there an extended crypter?

i.e., why “base”?

Reply:
no, there won't be an extended crypter, but this will be considered the `default` crypter implementation of the JWE compliant spec. 

The `base` is technically there to distinguish the implementation name from the interface


shall I remove Base prefix?

Reply:
I do not think Base is descriptive. I also do not think you can have no prefix with the Legacy one coming.

Reply:
Base has been removed.. implementations will be called Crypter in both jwe and legacy.. packages will have aliases during import ie: jweauthcrypt.Crypter, legacyauthcypt.Crypter

Reply:
done